setup.py

```diff
@@ -55,7 +55,7 @@
         "graph": ["networkx", "matplotlib"],
         "websurfer": ["beautifulsoup4", "markdownify", "pdfminer.six", "pathvalidate"],
         "redis": ["redis"],
-        "ipython": ["jupyter-client>=8.6.0", "ipykernel>=6.29.0"],
+        "ipython": ["jupyter_kernel_gateway", "websocket-client", "requests"],
```

Comment:
We might have to change this dependency to "jupyter"

Reply:
Yeah good call - however this dependency list is specific to local jupyer and python ill make it specific